plugin/aws/ec2Instance.go

```diff
@@ -264,8 +264,10 @@ func (inst *ec2Instance) Exec(ctx context.Context, cmd string, args []string, op
 		hostname = name.(string)
 	} else if ipaddr, ok := meta["PublicIpAddress"]; ok && ipaddr != nil {
 		hostname = ipaddr.(string)
+	} else if ipaddr, ok := meta["PrivateIpAddress"]; ok && ipaddr != nil {
+		hostname = ipaddr.(string)
```

Comment:
I think the one thing I want to add is an `activity.Record` message to help identify why the connection might be failing. I can add that when I get this merged.

Reply:
https://github.com/puppetlabs/wash/pull/512